Various bug fixes

kevin-10 opened the pull request.

kevin-10 committed:
*Requires Core Recompile*

-Fixed issue where client helpers could bypass the container gump and add items directly to the seed box
-SearedFireAntGoo is no longer stackable
-Blue NPC's (mainly vendors, not initial innocent) can no longer be attacked on trammel maps
-Aggressors/Aggressed from pet masters now works properly. This was suspected to crated large numbers to redundant aggressor/aggressed added to a mobiles aggressor/aggressed lists
-Aggressive action for healing a mobile while fighting another mobile has been re-enabled
-AuraDamage is now a harmful act, ie those affected will aggro on your pet!
-MLDryads should no longer attack other wild creatures during Blackthorn Invasion Instance

kevin-10 committed:
Same fix for scroll books

kevin-10 committed:
Merge branch 'master' into VariousBugFixes

DocKnight86 committed:
Merge branch 'master' into VariousBugFixes

TrueUO commented:
Two of the code smells could be fixed.

The other two about cognitive complexity, not much we can do about that in a lot of situations. While it would be nice to abide by the "15" rule, it can't always be possible.

DocKnight86 committed:
Update Notoriety.cs

https://sonarcloud.io/project/issues?id=trueuo&open=AXGeCoF4heqsK8FbU3CA&pullRequest=16&resolved=false&types=CODE_SMELL
